consumer.js

```diff
@@ -0,0 +1,14 @@
+"use strict"
```

Comment:
So perhaps you can help me out here, 2 things with injecting this into the main file:

1. Seneca will look locally for a pattern match (only took me like a day to figure this problem out), so the Seneca.act({role:updater,info:update}) will call onNpmChange, instead of going to redisQ. So I change the pattern, BUT

2. We have a recursive relationship going with the queue. 

So putting this in the main file:

```
  seneca.add('role:updater,info:update',  onNpmChange)

```

and then calling `seneca.act('role:updater,info:update') ` when we have a change or new package incoming, will place a package on the queue

```
 .client({pin:'role:updater,info:update',type:'redis-queue'})
```
When something comes off the queue, redis-queue will transmit the pattern 

`'role:updater,info:update'`

and because we are listening in the same place we are emitting, that package is going to end back up on the queue again

```
.listen({pin:'role:updater,info:update',type:'redis-queue'})

```

So the best I could come up with was to split it. I was thinking that maybe the listener belonged somewhere else altogether, but then if we end up with coupling the more I think of it.